docs/source/installation.rst

```diff
@@ -146,3 +146,68 @@ searching failed or you want to switch them:
     -DCUDA_TOOLKIT_ROOT_DIR=/path/to/cuda \
     -DCUDNN_ROOT_DIR=/path/to/cuda \
     -DPRIMITIV_USE_CUDA=ON
+
+
+Installing primitiv with OpenCL
+-------------------------------
+
+
+OpenCL backend requires OpenCL C++ headers (CLHPP) and CLBlast.
```

Comment:
Probably not necessary. Prerequisites are already documented in previous sections.